app/ide-desktop/lib/dashboard/src/authentication/src/dashboard/components/templates.tsx

```diff
@@ -1,6 +1,10 @@
 /** @file Renders the list of templates that can be used to create a project from. */
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 import * as React from 'react'
+/** We are using temporary background images. */
+import BgUrl1 from '../../../assets/TempTemplateBg1.jpg'
+import BgUrl2 from '../../../assets/TempTemplateBg2.jpg'
```

Comment:
Do not contribute binary files to the repo. This increases the repo size and is just a bad idea in general. We can't store binary data neither in repo, nor hardcoded in the code. Use plain colors for these divs until you got the final solution and these images will be delivered in the package, e.g. by the build system.

Reply:
Okay, I will use black as the background color.